{{cookiecutter.repo_name}}/travis_pypi_setup.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Update encrypted deploy password in Travis config file
+"""
+
+
+from __future__ import print_function
+import base64
+import json
+import os
+from getpass import getpass
+import yaml
+from cryptography.hazmat.primitives.serialization import load_pem_public_key
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric.padding import PKCS1v15
+
+
+try:
+    from urllib import urlopen
+except:
+    from urllib.request import urlopen
+
+
+GITHUB_REPO = '{{ cookiecutter.github_username }}/{{ cookiecutter.repo_name }}'
+TRAVIS_CONFIG_FILE = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), '.travis.yml')
+
+
+def encrypt(pubkey, password):
```

Comment:
Could you please add docstrings to this and the other functions, to help me understand better? :)